.github/workflows/pull-request.yaml

```diff
@@ -0,0 +1,41 @@
+name: PR Check
+
+on: [pull_request]
+
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Build (Linux)
+      run: make build
+
+    - name: Upload binaries
+      uses: actions/upload-artifact@v2
+      with:
+        name: opa-docker-authz
+        path: ./
```

Comment:
If we want to upload just one binary, we should do that... I think `opa-docker-authz` is the collective "artifact name", and with `path: ./` we'll perhaps upload the entire working directory...? I could be wrong 🤔

Reply:
I think you're right. Might be excessive to upload the binary on each PR too.. I'll remove this for now.